kokoro/builds/build.sh

```diff
@@ -142,6 +142,13 @@ if [ -n "$1" ]; then
   echo "Starting the build (conan build)."
   conan build -bf "${REPO_ROOT}/build/" "${REPO_ROOT}"
 
+  if [[ $CONAN_PROFILE == "coverage_clang9" ]]; then 
+    echo "Starting to generate unit test coverage report"
+    mkdir -p "${REPO_ROOT}/build/package/coverage"
+    ${REPO_ROOT}/contrib/unit_test_coverage/generate_coverage_report.sh \
+            "${REPO_ROOT}/src" "${REPO_ROOT}/build" "${REPO_ROOT}/build/package/coverage"
+  fi
+
   if [[ $CONAN_PROFILE != "iwyu" ]]; then
```

Comment:
I think we don't want to call `conan package` when using the `coverage_clang9`, just like include-what-you-use.

If we did, we would store all the binaries in the bucket, again. There is another job doing that already.

Reply:
The same applies to the ZIP file generation in line 206. This has to be exluded as well.

Reply:
One more thing. Could you put this part further down in the script? Maybe after the `Analyze include-what-you-use` results?

Reply:
I fixed this and tested that it works.